lib/app_prototype/repo.ex

```diff
@@ -1,3 +1,11 @@
 defmodule AppPrototype.Repo do
   use Ecto.Repo, otp_app: :app_prototype
+
+  @doc """
+  Dynamically loads the repository url from the
+  DATABASE_URL environment variable.
+  """
+  def init(_, opts) do
+    {:ok, Keyword.put(opts, :url, System.get_env("DATABASE_URL"))}
```

Comment:
This is new Phoenix convention, allowing us to pick up the new config at runtime and avoiding compile-time issues.